examples/hello_triangle_c/CMakeLists.txt

```diff
@@ -3,13 +3,20 @@ cmake_minimum_required(VERSION 3.11b)
 project(hello_triangle)
 
 set(TARGET_NAME hello_triangle)
-
+set(feature-native)
 add_executable(hello_triangle main.c)
 
 if(MSVC)
     add_definitions(-DWGPU_TARGET=WGPU_TARGET_WINDOWS)
     target_compile_options(${TARGET_NAME} PRIVATE /W4)
     set(GLFW_LIBRARY glfw3)
+
+    target_link_libraries(${TARGET_NAME} userenv ws2_32 Dwmapi)
```

Comment:
We also need to link `dbghelp` here for debug builds -- it looks like `backtrace` relies on it.

Reply:
I don't know that about `dbghelp`, I could include it no problem, but do you experience problems without it?

Reply:
Without it, I get a few `backtrace`-related linker errors:

```
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol SymInitializeW referenced in function _ZN9backtrace12dbghelp_init17h846b2405be6646eaE 
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol SymFromAddrW referenced in function _ZN9backtrace9symbolize7dbghelp7resolve17h87c86220145b776bE
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol SymGetLineFromAddrW64 referenced in function _ZN9backtrace9symbolize7dbghelp7resolve17h87c86220145b776bE
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol SymFunctionTableAccess64 referenced in function _ZN9backtrace9backtrace20trace_unsynchronized17he6ede2ba53fdf73fE
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol SymGetModuleBase64 referenced in function _ZN9backtrace9backtrace20trace_unsynchronized17he6ede2ba53fdf73fE
wgpu_native.lib(backtrace-[...].o) : error LNK2019: unresolved external symbol StackWalk64 referenced in function _ZN9backtrace9backtrace20trace_unsynchronized17he6ede2ba53fdf73fE
C:\[...]\hello_triangle.exe : fatal error LNK1120: 6 unresolved externals
```

Reply:
Oh I have not had those errors. I will look into it.